src/bin/common/solving_loop.ml

```diff
@@ -182,14 +224,15 @@ let main () =
       with
       | Util.Timeout ->
         FE.print_status (FE.Timeout None) 0;
-        exit 142
+        exit_as_timeout ()
       | Parsing.Parse_error ->
-        Printer.print_err "%a" Errors.report
-          (Syntax_error ((Lexing.dummy_pos,Lexing.dummy_pos),""));
-        exit 1
+        (* TODO: display a better error message *)
```

Comment:
This is error messages for the legacy parser — the way to have better error messages is to use the Dolmen frontend (which we want people to use anyways).

Reply:
Good, that means this comment will disappear when we switch to dolmen ;) In the meantime, there is explicitely in the code the printing of a dummy value: we should either remove it or document it. If we decide for any reason not to remove the legacy parser, it is better to have some documentation about it. I will make it more explicit.

Reply:
Yeah, being more explicit with the comment (or even just not using `Errors.report` here because it doesn't really make sense) is OK but it would be a waste to dig too deep into this (unless we ultimately decide not to switch to Dolmen, as you say).

Now that I think about it -- the plan is to make Dolmen the default frontend for the next release, which means we should make it the default on `next` right now so that we can iron out the issues. I will make a PR to do so.